site/oauth2.md

```diff
@@ -314,6 +315,23 @@ In the OAuth context, tags can be added as part of the scope, using a format lik
 example, if `resource_server_id` is "my_rabbit", a scope to grant access to the management plugin with
 the `monitoring` tag will be `my_rabbit.tag:monitoring`.
 
+### <a id="preferred-username-claims" class="anchor" href="#preferred-username-claims">Preferred username claims</a>
+
+RabbitMQ needs to figure out the username associated to the token so that it can display it in the management ui.
+By default, RabbitMQ will first look for the `sub` claim and if it is not found it uses the `client_id`.
+
+Most authorization servers return the user's GUID in the `sub` claim rather than the actual user's username or email address, anything the user can relate to. When the `sub` claim does not carry a *user-friendly username*, you can configure one or several claims to extract the username from the token.
```

Comment:
" anything the user can relate to."....what does this part of the sentence mean in this context? should the word the word "or" be added before this text then it would makes sense???